tcpclient/connectionsManagement.go

```diff
@@ -8,10 +8,12 @@ import (
 	"strconv"
 	"sync"
 	"time"
+	"reflect"
 )
 
-// this can be an argument in the future
-var defaultDialTimeoutInSecs = 10
+// this first one can be a flag in the future
+var DefaultDialTimeoutInSecs = 5
```

Comment:
why is it exportable being only used in this package?

Reply:
I think you were talking a look to an old version. Now yes, its being consumed as main is exposing this as a flag